test/Test/Chapter1QC.hs

```diff
@@ -0,0 +1,13 @@
+{-# OPTIONS_GHC -Wno-type-defaults #-}
+
+module Test.Chapter1QC
+    ( firstDigitRevLastDigit
+    ) where
+
+import Chapter1
+
+firstDigitRevLastDigit :: Int -> Bool
+firstDigitRevLastDigit x = firstDig == lastDig
+    where
+      lastDig = lastDigit (read  (reverse $ show $ abs x) :: Int)
```

Comment:
I think in here the sign should be preserved as well for the more fair property 🙂